src/lib/helpers/config.ts

```diff
@@ -12,6 +12,13 @@ const ConfigSchema = z.object({
         encrypt_query_params: z.boolean().default(
             Deno.env.get("SERVER_ENCRYPT_QUERY_PARAMS") === "true" || false,
         ),
+        enable_metrics: z.boolean().default(
+            Deno.env.get("SERVER_ENABLE_METRICS") === "true"
+                ? true
+                : Deno.env.get("SERVER_ENABLE_METRICS") === "false"
```

Comment:
Why do you check for enable metrics env to be false?

If you remove the parameter it's false by default.

Reply:
yeah, default of `Deno.env.get("SERVER_ENABLE_METRICS") === "true"` will make it a boolean correctly. At the moment it'd always be true unless the env variable `SERVER_ENABLE_METRICS` is `"false"`